src/main/java/dat/backend/model/persistence/LinkItemListMaterialsVariantsMapper.java

```diff
@@ -0,0 +1,34 @@
+package dat.backend.model.persistence;
+
+import dat.backend.model.entities.LinkItemListMaterialVariants;
+import dat.backend.model.exceptions.DatabaseException;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class LinkItemListMaterialsVariantsMapper {
+
+    static void addLink(int itemListId, int materialVariantsId, ConnectionPool connectionPool) throws  DatabaseException{
+
+        String sql = "INSERT INTO item_list (item_list_id, material_variants_id) VALUES (?,?)";
```

Comment:
Change SQL string to this
"INSERT INTO link_item_list_material_variants (item_list_id, material_variants_id) VALUES (?,?)"